### PR TITLE
doc(theme): update links to access components

### PR DESCRIPTION
**Pull request by slemouillour**
**What is the problem this PR is trying to solve?**
1- one link in navbar is broken 
2- after click in menu (#banner), the list of link is not accessible
**What is the chosen solution to this problem?**
1- i have deleted it
2- add a dropdown-menu in the navbar with all links when the bloc "banner" is not visible on screen 
**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/example/index.html [hunk 1]
```diff
@@ -1629,6 +1640,22 @@ <h4 class="modal-title">Source Code</h4>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script>
     (function(){
+        function setVisibilyDropDownMenu(){
+          var top = $(document).scrollTop();
+          var elementTop = $("#banner").offset().top;
+          var elementBottom = elementTop + $("#banner").height();
+
+          if ((top < elementTop) || (top < elementBottom)) {
```
Comment (jsomsanith) [inline packages/theme/example/index.html]: `top < elementBottom` should be sufficient.

File: packages/theme/example/index.html [hunk 2]
```diff
@@ -1629,6 +1640,22 @@ <h4 class="modal-title">Source Code</h4>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script>
     (function(){
+        function setVisibilyDropDownMenu(){
+          var top = $(document).scrollTop();
+          var elementTop = $("#banner").offset().top;
+          var elementBottom = elementTop + $("#banner").height();
```
Comment (frassinier) [inline packages/theme/example/index.html]: Please factorize:

```javascript 
var top = $(document).scrollTop();
var $banner = $("#banner");
var elementTop = $banner.offset().top;
var elementBottom = elementTop + $banner.height();
```

File: packages/theme/example/index.html [hunk 3]
```diff
@@ -1629,6 +1640,22 @@ <h4 class="modal-title">Source Code</h4>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script>
     (function(){
+        function setVisibilyDropDownMenu(){
```
Comment (frassinier) [inline packages/theme/example/index.html]: `setDropDownMenuVisibility` instead?

File: packages/theme/example/index.html (hunk 1 above)
Comment (frassinier) [reply packages/theme/example/index.html]: You can use [toggle fn](http://api.jquery.com/toggle/#toggle-display) instead like:

```javascript
$('#menu_dropdown').toggle(top < elementBottom);
```

File: packages/theme/example/index.html [hunk 4]
```diff
@@ -1629,6 +1640,22 @@ <h4 class="modal-title">Source Code</h4>
...
+          var elementBottom = elementTop + $("#banner").height();
+
+          if ((top < elementTop) || (top < elementBottom)) {
+            $('#menu_dropdown').hide();
+          } else {
+            $('#menu_dropdown').show();
+          }
+        }
+
+        $(document).ready(function(){
```
Comment (frassinier) [inline packages/theme/example/index.html]: `$(function(){ /* ... */ }` should be enough

File: packages/theme/example/index.html [hunk 5]
```diff
@@ -30,12 +30,23 @@
...
+                <li><a href="#breadcrumbs">Breadcrumbs</a></li>
+                <li><a href="#indicators">Indicators</a></li>
+                <li><a href="#progress-bars">Progress bars</a></li>
+                <li><a href="#containers">Containers</a></li>
+                <li><a href="#dialogs">Dialogs</a></li>
+              </ul>
+            </li>
           </ul>
-
-          <ul class="nav navbar-nav navbar-right">
```
Comment (frassinier) [inline packages/theme/example/index.html]: Why not tryin to fix SassDoc generation instead? :thinking:

**Pull request merged**